frontend/lib/identification/verification_workflow/dialogs/negative_verification_result_dialog.dart

```diff
@@ -15,7 +15,7 @@ class NegativeVerificationResultDialog extends StatelessWidget {
       title: t.identification.notVerified,
       icon: Icons.error,
       iconColor: Colors.red,
-      child: Text(reason),
+      child: Text(reason, style: Theme.of(context).textTheme.bodyLarge),
```

Comment:
The content is bigger than the title. Can we apply some style to the title as well?
![image](https://github.com/user-attachments/assets/20b376c7-e509-4881-9204-27e51d090b24)